docs/content/docs/projects/CiSteps.md

```diff
@@ -1,52 +1,68 @@
 ---
-title: CI Gradle Plugin
+title: CI steps plugin
 type: docs
 ---
 
-# CI Gradle plugin
+# CI steps plugin
 
-Для настройки CI под конкретный Gradle модуль используется in-house плагин.
+{{< hint warning>}}
+Plugin wasn't tested outside Avito yet, so expect difficulties, or even blockers.
+However, if you interested, please [contact us]({{< ref "/docs/Contacts.md" >}})!
+{{< /hint >}}
```

Comment:
I think it would be good to have such common disclaimer in all plugins to explain a status.

Reply:
don't know how to use hints and refs inside shortcodes
` template: shortcodes/plugin-not-ready.html:2: unexpected "<" in command`

Reply:
Let's do it in another issue.

Reply:
ok; found only solutions with a lot of coding around :(